Vagrantfile

```diff
@@ -2,10 +2,19 @@
 # vi: set ft=ruby :
 
 Vagrant.configure("2") do |config|
-  config.vm.box = "lucid64"
-  config.vm.box_url = "http://files.vagrantup.com/lucid64.box"
+  config.vm.box = "ubuntu/trusty64"
```

Comment:
Ah, I modified it because for whatever reason the chef scripts didn't work for me so I simplified it to just use apt-get to gather dependencies. If they do still work correctly for you I can back that out.